geolocator_windows/pubspec.yaml

```
@@ -0,0 +1,28 @@
+name: geolocator_windows
+description: Geolocation Windows plugin for Flutter. This plugin provides the Windows implementation for the geolocator.
+version: 0.1.0
+repository: https://github.com/Baseflow/flutter-geolocator/tree/master/geolocator_windows
+issue_tracker: https://github.com/Baseflow/flutter-geolocator/issues?q=is%3Aissue+is%3Aopen
+
+environment:
+  sdk: ">=2.12.0 <3.0.0"
+  flutter: ">=1.20.0"
```

As we are in the progress updating the requirements of the other packages at the moment we might as well use latest settings here:

```suggestion
  sdk: ">=2.15.0 <3.0.0"
  flutter: ">=2.8.0"
```